t/27-consoles-s3270.t

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/perl
+
+use Test::Most;
+use Mojo::Base -strict, -signatures;
+use Test::Mock::Time;
+use Test::MockModule 'strict';
+use Test::MockObject;
+use Test::Output qw(combined_like stderr_like);
+use Test::Warnings qw(:all :report_warnings);
```

Comment:
If we are not using any functions to check certain warnings, we don't need to import them:
```suggestion
use Test::Warnings qw(:report_warnings);
```